cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js

```diff
@@ -45,24 +45,24 @@ describe('ui-data-import', () => {
         locationName: 'Annex (KU/CC/DI/A)',
         locationQuantityPhysical: '1',
         locationQuantityElectronic: '1' },
-      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.order,
+      actionProfile: { typeValue: FOLIO_RECORD_TYPE.ORDER,
         name: `C380446 Test P/E mix open order with instance, holdings, item ${getRandomPostfix()}` }
     },
     {
       mappingProfile: { name: `C380446 Create simple holdings for open order ${getRandomPostfix()}`,
-        typeValue : NewFieldMappingProfile.folioRecordTypeValue.holdings,
+        typeValue: FOLIO_RECORD_TYPE.HOLDINGS,
         permanentLocation: '"Main Library (KU/CC/DI/M)"',
         permanentLocationUI:'Main Library' },
-      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.holdings,
+      actionProfile: { typeValue: FOLIO_RECORD_TYPE.HOLDINGS,
         name: `C380446 Create simple holdings for open order ${getRandomPostfix()}` }
     },
     {
       mappingProfile: { name: `C380446 Create simple item for open order ${getRandomPostfix()}`,
-        typeValue : NewFieldMappingProfile.folioRecordTypeValue.item,
+        typeValue: FOLIO_RECORD_TYPE.ITEM,
         materialType:'book',
```

Comment:
this value can be taken from the constant

Reply:
it is fixed in pr 1133